internal/cli/import/config.go

```diff
@@ -1,8 +1,29 @@
 package cli
 
-// Put here the logic for parsing the JSON config file
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+)
 
 type Config struct {
 	Auth0Domain                 string   `json:"AUTH0_DOMAIN"`
 	Auth0KeywordReplaceMappings struct{} `json:"AUTH0_KEYWORD_REPLACE_MAPPINGS"`
```

Comment:
This should probably be a `map[string]interface{}`, because it can contain any valid JSON value (e.g. numbers, strings). See https://auth0.com/docs/deploy-monitor/deploy-cli-tool/environment-variables-and-keyword-mappings, e.g.
```json
"AUTH0_KEYWORD_REPLACE_MAPPINGS": {
  "ENVIRONMENT_URL": "http://dev.travel0.com",
  "JWT_TIMEOUT": 120,
  ...
}
```